packages/appeals-service-api/src/routes/v2/appeal-cases/controller.js

```diff
@@ -21,7 +21,7 @@ async function getByCaseReference(req, res) {
 	}
 	try {
 		// only check published cases
-		const appealCase = await getCaseAndAppellant({ caseReference, casePublished: true });
```

Comment:
I believe this is always published so have removed from being passed around everywhere